.github/workflows/gen.yaml

```diff
@@ -0,0 +1,28 @@
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  test:
```

Comment:
Can you change the name of this job to `generate`, and we can merge?

Reply:
done in [fe5fac6](https://github.com/gnolang/tx-indexer/pull/55/commits/fe5fac6f41f573b53ad0063ebabff73e5c9317b6)